apps/files/src/default.js

```diff
@@ -56,6 +56,14 @@ const navItems = [
       path: `/${appInfo.id}/favorites`
     }
   },
+  {
+    name: 'Shared with others',
```

Comment:
we need to find a way to get this translated ....

Reply:
I guess we could translate it inside the menu component in computed properties but doesn't really seem like a nice solution.